src/ophys_etl/workflows/on_prem/dags/cell_classification/roi_classifier_training.py

```diff
@@ -122,35 +160,45 @@ def _generate_correlation_projections_for_experiments(
 )
 def roi_classifier_training():
     @task_group
-    def generate_correlation_projections():
+    def generate_correlation_projections(experiment_ids):
```

Comment:
Type hint?

Reply:
value is xcom, type unknown until runtime